drivers/led/lp5562.c

```diff
@@ -41,6 +41,9 @@ LOG_MODULE_REGISTER(lp5562);
 
 #include "led_context.h"
 
+#define DEV_CFG(dev)  ((const struct lp5562_config *)(dev->config))
+#define DEV_DATA(dev) ((struct lp5562_data *)(dev->data))
```

Comment:
please don't use these type of accessors, they were dropped a while ago from all drivers upstream.